### DISCUSSION TITLE
Anomalies

### DISCUSSION THREAD

**Discussion by apex944** [mivano/azure-cost-cli#171] [Q&A]
Looks like there is a missing word after "different" ?

`The console output will render the different anomalies in _different_ and point out the moment in time where the anomaly was detected.`

**Comment by apex944** [comment]
Another question...
> --recent-activity-days

does changing this value only change it for the command as entered once?  Or for the session, or ? 
IOW, does it default back to 7 immediately if not specified?

**Reply by mivano** [reply]
There is no state stored. So you need to pass the arguments again for the next run.

**Comment by mivano** [comment]
> Looks like there is a missing word after "different" ?
> 
> `The console output will render the different anomalies in _different_ and point out the moment in time where the anomaly was detected.`

Ah yeah that's strange. I think I mean categories here. Adjusted the text. Thanks!
